### PR TITLE
Control Center and LDAP Auth, no RBAC

### PR DESCRIPTION
**Pull request by MosheBlumbergX**
Added scenario for Control Center and LDAP Auth, no RBAC

LDAP server was added groups and couple of users.

### CODE REVIEW

**Comment by MosheBlumbergX** [comment]
@rohit2b @xnulinu

File: security/plaintext-ldap-auth-control-Center/README.rst [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+#. Browse to Control Center:
+
+   ::
+   
+     http://localhost:9021
+
+
+#. Users: 
+
+    Full Control: Username:Jame Password:james-sceret
```
Comment (rohit2b) [inline security/plaintext-ldap-auth-control-Center/README.rst:218]: is this `james` and `james-secret` ?

File: security/plaintext-ldap-auth-control-Center/README.rst [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+   ::
+   
+     http://localhost:9021
+
+
+#. Users: 
+
+    Full Control: Username:Jame Password:james-sceret  
+
+    Restricted Control: Username:alice Password:alice-sceret
```
Comment (rohit2b) [inline security/plaintext-ldap-auth-control-Center/README.rst:220]: is this `alice-secret` ?

File: security/plaintext-ldap-auth-control-Center/README.rst [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+
+Shut down Confluent Platform and the data:
+
+::
+
+  kubectl delete -f $TUTORIAL_HOME/producer-app-data.yaml --namespace=confluent
+
+::
+
+  kubectl delete -f $TUTORIAL_HOME/confluent-platform-ccc-ldap.yamll --namespace=confluent
```
Comment (rohit2b) [inline security/plaintext-ldap-auth-control-Center/README.rst:236]:
```suggestion
  kubectl delete -f $TUTORIAL_HOME/confluent-platform-ccc-ldap.yaml --namespace=confluent
```

**Review by rohit2b — CHANGES_REQUESTED** [review]
Thanks for putting this together! I went through the scenario, and suggested a few fixes.

**Comment by MosheBlumbergX** [comment]
Thanks @rohit2b , much appreciated !

File: security/plaintext-ldap-auth-control-Center/README.rst [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+#. Browse to Control Center:
+
+   ::
+   
+     http://localhost:9021
+
+
+#. Users: 
+
+    Full Control: Username:james Password:-secret
```
Comment (rohit2b) [inline security/plaintext-ldap-auth-control-Center/README.rst:218]:
```suggestion
    Full Control: Username:james Password:james-secret  
```

**Review by rohit2b — CHANGES_REQUESTED** [review]
One more fix, then done!

**Review by rohit2b — APPROVED** [review]
I made the one tiny fix. This is good to go!

**Pull request merged**